test/mocha/unicode.js

```diff
@@ -102,7 +102,7 @@ describe("Unicode", function() {
         it("Should not unescape unpaired surrogates", function() {
             this.timeout(20000);
             var code = [];
-            for (var i = 0; i <= 0x20001; i++) {
+            for (var i = 0; i <= 0x20001; i+=100) {
```

Comment:
Likewise please back out all changes to `test/mocha/unicode.js`.

This is an extremely important test for minification integrity and has caught many bugs.

Reply:
Reverted both of these